shell/browser/notifications/notification.h

```diff
@@ -53,10 +53,11 @@ class Notification {
   // Closes the notification, this instance will be destroyed after the
   // notification gets closed.
   virtual void Dismiss() = 0;
+  virtual void Remove();
```

Comment:
There should be comment clarifying the difference between `Dismiss` and `Remove`.